drivers/user_permission_report/app/controllers/user_permission_report/warehouse_reports/reports_controller.rb

```diff
@@ -40,5 +46,45 @@ def associated_items(user, meth)
       }
     end
     helper_method :associated_items
+
+    def hmis_access_report_rows
```

Comment:
this is fine for now as it follows the existing pattern, but it sure feels like this is getting big (and important) enough that we should build out a report class to hold the logic.  What do you think?

Reply:
I also opened https://github.com/open-path/Green-River/issues/7483 for updating the warehouse user permission sheet to support ACLs, so maybe when we do that we can move that logic into the class as well

Reply:
`AccessLogs::Report` looks like a great starting point.